tm/src/main/java/io/seata/tm/api/TransactionalTemplate.java

```diff
@@ -143,7 +144,9 @@ private void rollbackTransaction(GlobalTransaction tx, Throwable ex) throws Tran
         tx.rollback();
         triggerAfterRollback();
         // 3.1 Successfully rolled back
-        throw new TransactionalExecutor.ExecutionException(tx, TransactionalExecutor.Code.RollbackDone, ex);
+        throw new TransactionalExecutor.ExecutionException(tx,
```

Comment:
I feel that the code you changed is not correct. I think it is OK to throw RollbackDone directly here.